recipes/jthread-lite/all/conanfile.py

```diff
@@ -35,3 +36,6 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package_multi"] = "nonstd"
         self.cpp_info.components["jthreadlite"].names["cmake_find_package"] = "jthread-lite"
         self.cpp_info.components["jthreadlite"].names["cmake_find_package_multi"] = "jthread-lite"
+        if not self.settings.os == "Windows":
+            self.cpp_info.components["jthreadlite"].cxxflags = ["-pthread"]
```

Comment:
Since Conan supports things like Android and watchOS, let's be more conservative... The compiler flag is usually not required

```suggestion
        if not self.settings.os in ["Linux", "FreeBSD"]:
```

Reply:
Thanks  @prince-chrismc . Pushed a change without the 'not'.